mcpy/box.py

```diff
@@ -1,8 +1,9 @@
 import numpy as np
 
+
 class Box:
-    def __init__(self, box_dim):
-        self.box_dim = box_dim
+    def __init__(self, box_dims):
+        self.box_dims = box_dims
 
     def volume(self):
```

Comment:
I m considering if we could change the box.volume() as a method to property box.volume, it can be done at a later stage.

Reply:
@bdnguye2 can you make volume a property and change it in the `mcpy.integrator` class methods.

Reply:
I adjusted the box.py using the @property decorator under the volume function. I'll push the changes soon.